client/payments/data/types.ts

```diff
@@ -0,0 +1,32 @@
+export type PaymentGateway = {
+	id: string;
+	title: string;
+	description: string;
+	order: number | string;
+	enabled: boolean;
+	method_title: string;
+	method_description: string;
+	method_supports: Array< string >;
```

Comment:
This can be `string[]`

Reply:
Updated all changes in https://github.com/woocommerce/woocommerce-admin/commit/5687a413358e711567e00fd0579a87fcd9897d13